controllers/reconcilers/kserve_authconfig_reconciler.go

```diff
@@ -18,35 +18,34 @@ package reconcilers
 import (
 	"context"
 	"fmt"
-
 	"github.com/go-logr/logr"
 	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
 	authorinov1beta2 "github.com/kuadrant/authorino/api/v1beta2"
+	"github.com/kuadrant/authorino/pkg/log"
 	"github.com/opendatahub-io/odh-model-controller/controllers/comparators"
 	"github.com/opendatahub-io/odh-model-controller/controllers/constants"
 	"github.com/opendatahub-io/odh-model-controller/controllers/processors"
 	"github.com/opendatahub-io/odh-model-controller/controllers/resources"
 	k8serror "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+var _ SubResourceReconciler = (*KserveAuthConfigReconciler)(nil)
+
 type KserveAuthConfigReconciler struct {
```

Comment:
`KserveAuthConfigReconciler` should implement `NoResourceRemoval` interface as it have owner relationship with inference resource.

Reply:
See https://github.com/opendatahub-io/odh-model-controller/pull/185#discussion_r1551353696